### ISSUE TITLE
Some users are unable to see the contents of the retrospective board

### ISSUE THREAD

**Issue by lanerfy182** [microsoft/vsts-extension-retrospectives#291]
Users which have the same access with the ones having no issues are unable to see the contents of the retrospective board. 

![image](https://user-images.githubusercontent.com/55248717/141820634-c471d446-38f3-419b-8ffe-e05ffe46907d.png)

**Comment by eltjo** [comment]
I have the same issue, it worked in the previous version (1.70.5)

**Comment by RamboFIN** [comment]
We had a case where one user couldn't access any retros for any teams. 

TLDR Solution: Clear visit history
![image](https://user-images.githubusercontent.com/96041830/145765366-85eb427c-5c2a-438e-91c7-6b7c089d3f65.png)

Longer version rootcause: 
When the team took the retro extension into use, the person with this problem did a testrun with a test retro board, which was based on an different retro template.

The other test user removed the testboard after their testrun and then had created another retro board using a different template. 

Conclusion, the this "feature" happens if one user starts using one board and another user removes that board. 

Solution: reset visit history to clear the "cache"

Hope this helps in your issues as well.

**Comment by dsczltch** [comment]
Some of our users also have the same issue, they can't see the boards. 
This workaroud works but it would be preferable to fix this anoying issue.
Thank you!

**Comment by dphanson63** [comment]
This is an old issue.
